service_factory.go

```diff
@@ -110,7 +110,13 @@ func (f *ServiceFactory) prepareGateway(config *API, gatewayOpts *GatewayOptions
 	}
 
 	mux := http.NewServeMux()
-	mux.Handle("/", runtimeMux)
+	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
This method is generic for all gateway instance, I think the console specific change could be made in https://github.com/aserto-dev/topaz/blob/1bd29a25990c1963314032f4c5d61c70ac4e6837/pkg/app/console.go#L34C26-L34C48